### DISCUSSION TITLE
Respect wildignore when completing files

### DISCUSSION THREAD

**Discussion by gi1242** [L3MON4D3/LuaSnip#1057] [General]
Hi There,

Looks like LuaSnip completes *all* files during filename completion. Is it possible to configure it so that it respects `wildignore`? For instance I never want `.blg` or `.aux` files to show up in my completion...

Thanks

**Comment by L3MON4D3** [comment]
Is it possible you're confusing luasnip with cmp_luasnip?
I'm not sure where we complete any files :D

**Comment by bew** [comment]
Like @L3MON4D3 mentions, I think you're confusing `LuaSnip` with actually `cmp-path`, the `cmp` completion source for paths.
The missing behavior you're seeing [was already reported on `cmp-path` repo](https://github.com/hrsh7th/cmp-path/issues/56), if you want to get it fixed I'd suggest you to look there and contribute on that repo if you can.

**Reply by gi1242** [reply]
Thanks so much. This indeed is a `cmp-path` issue.
